src/components/inputs/localized-multiline-text-input/translation-input.styles.js

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/core';
+import vars from '../../../../materials/custom-properties';
+import { getInputStyles } from '../styles';
+
+/* we need this line-height to achieve 32px height when the component has only one row */
+const sizeInputLineHeight = '22px';
+
+// NOTE: order is important here
+// * a disabled-field currently does not display warning/error-states so it takes precedence
+// * a readonly-field cannot be changed, but it might be relevant for validation, so error and warning are checked first
+// how you can interact with the field is controlled separately by the props, this only influences visuals
+const getTextareaStyles = props => {
+  const baseStyles = [
+    getInputStyles(props),
```

Comment:
The "shared" input styled now can simply be spread, and overriden depending on the component